src/ColoredTabs/Item.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component, PropTypes } from 'react'
+import { View, Text, TouchableWithoutFeedback, StyleSheet } from 'react-native'
+
+export default class Item extends Component {
+  static propTypes = {
+    id: PropTypes.string,
+    active: PropTypes.bool,
+    color: PropTypes.string,
+    name: PropTypes.string,
+    onPress: PropTypes.func,
```

Comment:
The same as with "active" - we don't pas "onPress" to every item. So, if we make it required and pass single onPress to entire component, we have warning.